src/views/List.jsx

```diff
@@ -23,18 +24,37 @@ export function List({ data }) {
 		setSearchTerm('');
 	};
 
+	const renderedListLength = renderedList.length;
+
 	return (
 		<>
-			<form onSubmit={clearSearchField}>
-				<label htmlFor="search-filter">Item name:</label>
-				<input
-					type="text"
-					id="search-filter"
-					value={searchTerm}
-					onChange={(e) => setSearchTerm(e.target.value)}
-				/>
-				<button type="submit"> clear </button>
-			</form>
+			{renderedListLength === 0 && (
+				<>
+					<h2> Your list currently has no items. </h2>
+					<h3>Click on the add first item button to start your list.</h3>
+					<Link to="/add-item">
+						<button> add first item</button>
+					</Link>
+				</>
+			)}
+			{renderedListLength > 0 && (
+				<>
+					<form onSubmit={clearSearchField}>
+						<label htmlFor="search-filter">
+							Search for an item in your list:
+						</label>
+						<input
+							type="text"
+							id="search-filter"
+							value={searchTerm}
+							onChange={(e) => setSearchTerm(e.target.value)}
+						/>
+						<button type="submit"> clear to reset search </button>
+					</form>
+					<h3> Here are the items in your list:</h3>
+				</>
+			)}
+
```

Comment:
This is really elegant, nice job. I had no idea you could use an inline conditional that way.